webpack.config.js

```diff
@@ -0,0 +1,29 @@
+var path = require('path');
+
+module.exports = {
+	entry: './src/app.js',
+	output: {
+		filename: 'app.js',
+		path: path.resolve(__dirname, 'dist')
+	},
+	target: 'node',
+	module: {
+		loaders: [{
+			test: /\.js$/,
+			use: require.resolve('./remove-hashbang-loader')
+		}]
+	},
+	externals: {
+		'mkfifo': 'commonjs mkfifo',
+		'sqlite3': 'commonjs sqlite3',
+		'mysql2': 'null',
```

Comment:
The idea was to load this modules as `null` - but now I've changed to using null-loader (didn't know it existed, so thanks!)